README.md

```diff
@@ -20,4 +20,31 @@ jobs:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         teams: "my-github-team"         # only works for GitHub Organisation/Teams
         persons: "aditya"               # add individual persons here 
```

Comment:
```
          teams: "team1"
          persons: "???"
```

What is the format of persons: github-username or @github-username?

Reply:
Will modify